ts/features/wallet/onboarding/cobadge/screens/add-account/AddCobadgeComponent.tsx

```diff
@@ -79,25 +75,12 @@ const AddCobadgeComponent: React.FunctionComponent<Props> = (props: Props) => {
               IOStyles.horizontalContentPadding
             ]}
           >
-            <H1 style={styles.title}>{screenTitle}</H1>
+            <H2 style={styles.title}>{screenTitle}</H2>
             <VSpacer size={8} />
-            <H4 weight={"Regular"} style={styles.flexStart}>
-              {label}
-            </H4>
+            <H6 style={styles.flexStart}>{label}</H6>
             <VSpacer size={24} />
             <PreviewCoBadgeCard coBadge={props.pan} abi={abiInfo} />
             <VSpacer size={24} />
-            {isCoBadgeBlocked(props.pan) ? (
-              <InfoBox iconColor="red" iconName="errorFilled">
-                <Body>{blockedCard}</Body>
-              </InfoBox>
-            ) : (
-              <InfoBox>
-                <Body>
-                  {warning1} <Label color={"bluegrey"}>{warning2}</Label>
-                </Body>
-              </InfoBox>
-            )}
```

Comment:
Is it not needed anymore?

Reply:
@ChrisMattew Same as above. These screens will be deleted soon, so we can easily get rid of this legacy component and its instances scattered around the codebase.